EuchreSolution/Euchre/GameScript/card.ts

```diff
@@ -18,6 +18,27 @@ enum Rank {
 	Right = 16,
 }
 
+const suitTranslations = [
```

Comment:
Can we just change the order of the existing suits enum? I think that will just work (except maybe it will break some of your unit tests)

Reply:
Done